### ISSUE TITLE
Dev Container through SSH sometimes only shows "[Dev Container]" instead of "[Dev Container Image @ ssh://remote]" on Get Started page

### ISSUE THREAD

**Issue by amurzeau** [microsoft/vscode-remote-release#8578]
- VSCode Version:
Version: 1.79.0-insider
Commit: aee52551e25d98391f9064996196cd83a9fc8669
Date: 2023-06-06T15:03:06.210Z
Electron: 22.5.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.1.0-9-amd64

- Local OS Version: Debian GNU/Linux 12 (bookworm)
- Remote OS Version: Debian GNU/Linux bookworm/sid
- Remote Extension/Connection Type: Dev Containers + SSH and Dev Container only

Dev Container extension version : v0.294.0
Remote - SSH extension version : v0.103.2023060615

Steps to Reproduce:

1. Have 2 differents SSH hosts
2.On each SSH hosts, do this:
  1. Create a testdevcontainer folder
  2. In this folder, create a .devcontainer/devcontainer.json file containing this text:
```json
{
	"name": "Alpine",
	"image": "mcr.microsoft.com/devcontainers/base:alpine-3.16",
	"remoteUser": "root"
}
```
3. Open the first SSH host using `Remote Explorer`
4. Open the `testdevcontainer` folder using `Open Folder` button on the `Get Started` page
5. When asked to, click on `Reopen in Container`:
![image](https://github.com/microsoft/vscode-remote-release/assets/5435069/f6fcbd09-bb76-469c-bd39-f35bcad95558)

4. Close the Container
5. Do the same thing with the second SSH host

6. Close the Container
7. The `Get Started` page shows duplicated `testdevcontainer [Dev Container]`:
![image](https://github.com/microsoft/vscode-remote-release/assets/5435069/93e29c7e-f49e-46eb-95a9-06b1db495fab)

Instead, I'm expecting this instead of `testdevcontainer [Dev Container]`:
- `testdevcontainer [Dev Container Alpine @ ssh://debian64_2]`
- `testdevcontainer [Dev Container Alpine @ ssh://debian64]`

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: No

**Comment by bamurtaugh** [comment]
Thanks for opening!

When actually connected to each dev container, do you see `Dev Container Alpine @ ssh://debian64` and `Dev Container Alpine @ ssh://debian64_2` in the remote indicator?

**Comment by amurzeau** [comment]
Yes, I see the complete label in the remote indicator like `Dev Container: Alpine @ ssh://debian64`.

I've retried with latest insiders:

In bottom left remote indicator of `Dev Container Alpine @ ssh://local` folder:
![image](https://github.com/microsoft/vscode-remote-release/assets/5435069/622b31bd-2b6e-42db-97dd-9a9d630d9c5c)

In bottom left remote indicator of `Dev Container Alpine @ ssh://local_2` folder:
![image](https://github.com/microsoft/vscode-remote-release/assets/5435069/adba2860-ba15-436b-9a3e-b5a9ccf736f8)

In Get Started:
![image](https://github.com/microsoft/vscode-remote-release/assets/5435069/820ad31f-4702-4998-a8d1-bb30ad5fd444)

VSCode version:
Version: 1.80.0-insider
Commit: 636dbcff557af8aa088ae118b1a094dce6632ce7
Date: 2023-06-21T08:21:04.745Z
Electron: 22.3.13
ElectronBuildId: 21803499
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.3.0-1-amd64

**Comment by bamurtaugh** [comment]
Thanks so much for the great details!

The abbreviation in the getting started or recent area seems potentially intentional to me - tagging @chrmarti @bhavyaus for further thoughts

**Comment by amurzeau** [comment]
No more information was asked, so I guess this was wrongly closed automatically.
Can this issue be reopened ?

**Comment by bhavyaus** [comment]
This issue requires change to the reported label of Dev container workspaces. The getting started service simply reports the returned label here:

https://github.com/microsoft/vscode/blob/fe0632cbb2d6d3517676748a9601d3cb3dc36b01/src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.ts#L964-L965
